exercism/phone-number/phone-number.js

```diff
@@ -0,0 +1,34 @@
+var PhoneNumber = function(digits){
+  this.digits = digits
+}
+
+PhoneNumber.prototype.number = function() {
+
+  var answer = ''
+  var strippedNumber = this.digits.replace(/\D/g,'')
+
+  if (strippedNumber.length === 10) {
+    answer = strippedNumber
+  } else if (strippedNumber.length === 11 && strippedNumber.charAt(0) === '1') {
```

Comment:
remember strings can be also treated as arrays :) So if you are just trying to capture a certain character in a string then just use index